src/man.rs

```diff
@@ -349,15 +356,32 @@ fn env(page: Roff, environment: &[Env]) -> Roff {
 ///
 ///        2      Optional error
 /// ```
-fn exit_status(page: Roff) -> Roff {
-  page.section(
-    "EXIT STATUS",
-    &[
+fn exit_status(page: Roff, exit_statuses: &[ExitStatus]) -> Roff {
+  if exit_statuses.is_empty() {
+    return page;
+  }
+  let arr = if exit_statuses
+    .iter()
+    .any(|status| status.use_default_instead)
+  {
+    vec![
       list(&[bold("0")], &["Successful program execution.\n\n"]),
       list(&[bold("1")], &["Unsuccessful program execution.\n\n"]),
       list(&[bold("101")], &["The program panicked."]),
-    ],
-  )
+    ]
+  } else {
+    let mut arr = vec![];
+    for exit_status in exit_statuses {
+      let exit_code =
+        format!("{}", exit_status.code.expect("initialized with value"));
+      let mut exit_description =
+        String::from(exit_status.description.unwrap_or(""));
+      exit_description.push_str("\n\n");
+      arr.push(list(&[bold(&exit_code)], &[exit_description]));
+    }
+    arr
```

Comment:
This could probably be rewritten as `exit_statuses.map()`. More of a style thing tho, okay to leave it as-is also!